### PR TITLE
NDRS-531: Adjust accounts csv

### PR DESCRIPTION
**Pull request by mpapierski**
Ref: https://casperlabs.atlassian.net/browse/NDRS-531

`CONV_RATE` is changed to 1 as agreed.

Top gas usages of all deploys in the test suite (I ran it with `use-system-contracts` so it accomodates `standard_payment.wasm` for payment code at minimum) is measured at 1499920085980.

Starting balances in production/accounts.csv are already more than enough by ~133.33 times compared to accounts created during test suite run.

Average session code cost as measured by running the test suite is 7284098942 which with default production/accounts.csv balances gives ability to run average contract about 137285 times.

Most expensive payment code in the test suite (ran with `use-system-contracts`) is measured at `503151180`. Therefore I'm setting `MAX_PAYMENT` to 5x this value to `2500000000`

Reference file with payment/session code gas outputted through test suite ran with `use-system-contracts` https://gist.github.com/mpapierski/b884f6eb8b0c5d4c29ffceb0fefb3028

Example commands to play with the data:

```sh
# top payment code gas costs
awk -F, '/payment_gas_cost/ { print $2 }' all_gas_costs.csv | sort -nr | uniq | head
# top session code gas costs
awk -F, '/session_gas_cost/ { print $2 }' all_gas_costs.csv | sort -nr | uniq | head
```

### CODE REVIEW

**Comment by henrytill** [comment]
@mpapierski: you've got some conflicts here

File: resources/production/accounts.csv [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-01ae2af99944b0b9f92fccf425aa51894ebbad0f4e8e42c66a71dcb999a3bd94ed,1000000000000000000000000,0
-01bdda22721033ae556e92ccf1034f8d14f24c3794fa7befaa831bcae98f9f1e9b,1000000000000000000000000,0
-015ff157d012383e99a3ae93bed0586a23383eed41b0058e63f48ed7b82b067c34,1000000000000000000000000,0
-0123bf1bd107d82bb42f4a92fc1ce19251f341802e1ecfa6ed88a3c7bbbdc302d1,1000000000000000000000000,0
-01313a3192db07c50758138e9637faf0358081c2c8709f8a72372685baeaf2a8b0,1000000000000000000000000,0
-0144118517b5ba7e7ca917d5a418c5fb8bf49e279427435d6597f59f8c5bf9ff1f,1000000000000000000000000,0
-01aaa7deb9ec99d6bed089fc938cac4a77bc9539f53556af057f9351a1d3c81de9,1000000000000000000000000,0
-0186d42bacf67a4b6c5042edba6bc736769171ca3320f7b0040ab9265aca13bbee,1000000000000000000000000,0
\ No newline at end of file
+013f774a58f4d40bd9b6cce7e306e53646913860ef2a111d00f0fe7794010c4012,1000000000000000000000000,10000000
```
Comment (sacherjj) [inline resources/production/accounts.csv:1]:
```suggestion
01054c929d687267a30341c759b4a9cab1238cb2de65546be43dad479c50745724,1000000000000000000000000000,0
013f774a58f4d40bd9b6cce7e306e53646913860ef2a111d00f0fe7794010c4012,1000000000000000000000000,10000000
```

Comment (sacherjj) [reply resources/production/accounts.csv:1]: This adds in the faucet account for delta.

**Review by sacherjj — APPROVED** [review]
Looks good.  We can also add in faucet account if you like.  I need to get that in master.

**Review by AlexanderLimonov — APPROVED** [review]
Seems like enough zeros, I think

**Pull request merged**